.bazelrc

```diff
@@ -0,0 +1,4 @@
+# Force the tests to not run in sandboxed mode, since it can introduce
+# errors and flakes.
+test --spawn_strategy=local
+build --macos_minimum_os=10.9
```

Comment:
Can this not be specified on `swift_library`? Seems like something that one would want to set on a per-module basis.

Reply:
No, it's agnostic of OS.

Reply:
given you set this on the build tests do you actually need it? I guess the problem is the swift_test that you'd have to otherwise work around by having a swift_test for linux and macos_unit_test and an alias? that is a pain